agent/src/test/java/com/teamscale/jacoco/agent/commit_resolution/git_properties/GitPropertiesLocatorUtilsTest.java

```diff
@@ -0,0 +1,25 @@
+package com.teamscale.jacoco.agent.commit_resolution.git_properties;
+
+import org.assertj.core.api.Assertions;
+import org.junit.jupiter.api.Test;
+
+import java.io.File;
+import java.net.URL;
+
+class GitPropertiesLocatorUtilsTest {
+
+	@Test
+	public void parseSpringBootCodeLocations() throws Exception {
+		Assertions.assertThat(GitPropertiesLocatorUtils
+				.extractGitPropertiesSearchRoot(new URL("jar:file:/home/k/demo.jar!/BOOT-INF/classes!/")))
```

Comment:
Extract `/home/k/demo.jar` to a constant? It's used four times in this class AFAICS.

Reply:
I'd like to keep the tests independent. Changing one of them should not affect the others. And I find it much easier to read the test when the string is not chopped up into `"jar:file:" + TEST_JAR_PATH + "!/BOOT-INF/classes"`. I also don't think that renaming that file consistently across all tests is really a maintenance use case we should be optimizing for over the plain readability of the test.